packages/Material-Core.package/MDColorTheme.class/instance/calculateLightVariants..st

```diff
@@ -0,0 +1,10 @@
+color-calculating
+calculateLightVariants: aColor
+
+	| calculatedColors |
+	
+	calculatedColors := (aColor lightShades: self class numberOfShades + 2) asOrderedCollection.
+	calculatedColors remove: calculatedColors first;
```

Comment:
Das ```remove: calculatedColors first;``` sollte eingerückt in die Zeile drunter.

Auch hier ist nicht ganz klar, wo ```+2``` herkommt.